public/js/practise.js

```diff
@@ -25,14 +25,14 @@ function CompareNumberIfElse(number1, number2) {
 
     if (number1 > number2) {
 
-        result = number1;
+        result = number1 > number2;
 
         return "answer is " + number1;
     }
     else {
         (number2 > number1);
```

Comment:
You can remove here also line 33, because it has no meaning.
And you can remove line 35 because is useless.